lib/ui/screens/about.dart

```diff
@@ -0,0 +1,271 @@
+import 'package:cobble/localization/localization.dart';
+import 'package:cobble/ui/common/components/cobble_button.dart';
+import 'package:cobble/ui/common/components/cobble_card.dart';
+import 'package:cobble/ui/common/components/cobble_tile.dart';
+import 'package:cobble/ui/common/icons/fonts/rebble_icons.dart';
+import 'package:cobble/ui/common/icons/comp_icon.dart';
+import 'package:cobble/ui/router/cobble_scaffold.dart';
+import 'package:cobble/ui/router/cobble_screen.dart';
+import 'package:cobble/ui/theme/with_cobble_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:package_info/package_info.dart';
+import 'package:flutter_svg_provider/flutter_svg_provider.dart';
+import 'dart:io' show Platform;
+import 'package:intl/intl.dart';
+
+class About extends StatefulWidget implements CobbleScreen {
+  @override
+  State<StatefulWidget> createState() => _AboutState();
+}
+
+class _AboutState extends State<About> {
+  static const discordURL = "https://discord.gg/aRUAYFN";
+  static const helpCenterURL = "https://help.rebble.io/";
+  static const redditURL = "https://reddit.com/r/pebble";
+  static const emailURL = "mailto:support@rebble.io";
+  static const twitterURL = "https://twitter.com/pebble_dev";
+  static const sourceURL = "https://github.com/pebble-dev/mobile-app";
+  String _appVersion = '';
+  String _appName = '';
+  String os = toBeginningOfSentenceCase(Platform.operatingSystem) ?? '';
+
+  @override
+  void initState() {
```

Comment:
We use hooks in this project, please rewrite this using `useEffect` or `Future<void> initPlatformState+ FutureBuilder`.